### DISCUSSION TITLE
stories_things/basic_content/abstract/

### DISCUSSION THREAD

**Discussion by giscus** [Light-City/light-comments#18] [Announcements]
# stories_things/basic_content/abstract/

Stories About C Plus Plus

https://light-city.github.io/stories_things/basic_content/abstract/

**Comment by Tlntin** [comment]
这里看着没有同步，有些文件超链接失效了，得去：[https://github.com/Light-City/CPlusPlusThings/tree/master/basic_content/abstract](https://github.com/Light-City/CPlusPlusThings/tree/master/basic_content/abstract)看最新的。
